packages/features/timezone-buddy/components/AvailabilitySliderTable.tsx

```diff
@@ -143,6 +177,9 @@ export function AvailabilitySliderTable() {
     fetchMoreOnBottomReached(tableContainerRef.current);
   }, [fetchMoreOnBottomReached]);
 
+  // This means they are not apart of any teams so we show the upgrade tip
+  if (!flatData.length) return <UpgradeTeamTip />;
```

Comment:
Since this returns all members from found teams -> this will return an empty array if the user doesnt have any teams